tests/Unit/BCTest.php

```diff
@@ -284,6 +284,47 @@ public function shouldRound($expected, $number, $precision = 0): void
         self::assertSame($expected, BC::round((string)$number, $precision));
     }
 
+    public function roundHalfEvenProvider(): array
+    {
+        return [
+            ['2', '1.8'],
+            ['2', '1.5'],
+            ['1', '1.2'],
+            ['1', '0.8'],
+            ['0', '0.5'],
+            ['0', '0.2'],
+            ['-0', '-0.2'],
+            ['0', '-0.5'],
+            ['-1', '-0.8'],
+            ['-1', '-1.2'],
+            ['-2', '-1.5'],
+            ['-2', '-1.8'],
+            ['-2.35', '-2.35', 2],
+            ['-2.4', '-2.35', 1],
+            ['2.4', '2.35', 1],
+            ['0', '0.005', 2],
+            ['0.02', '0.015', 2],
+            ['0.02', '0.025', 2],
+            ['0.04', '0.035', 2],
+            ['0.04', '0.045', 2],
+            ['0.06', '0.055', 2],
+            ['0.06', '0.065', 2],
+            ['0.08', '0.075', 2],
+            ['0.08', '0.085', 2],
```

Comment:
Hi @aywan can you add bigger numbers for example:

`
            ['1234567890', '9876543210', '12193263111263526900', 2],
            ['2.5', '1.5', '3.75', 2],
            ['2.555', '1.555', '3.97', 2],
            [9.9999E-2, -5.6E-2, '-0.005599944', 9], 
```

you can use wolframalpha.com to confirm them 

tx

Reply:
Hi @krowinski 
Sorry, lost your comment. Can add it if necessary.

Reply:
yes pls